### PR TITLE
New package: zaz-1.0.0

### PR DESCRIPTION
**Pull request by pinnaculum**

### CODE REVIEW

**Comment by ndowens** [comment]
Will post how you can fix the cross-build after a while

File: srcpkgs/zaz/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'zaz'
+pkgname=zaz
+version=1.0.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="ftgl-devel SDL_image-devel libvorbis-devel libtheora-devel"
```
Comment (ndowens) [inline srcpkgs/zaz/template]: add freetype-devel to makedepends

File: srcpkgs/zaz/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+short_desc="Arcade action puzzle game"
+maintainer="David Ferlier <dispertio@gmx.es>"
+license="GPL-3.0-or-later"
+homepage="http://zaz.sourceforge.net"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.bz2"
+checksum=e332cc1a6559e18a2b632940c53d20e2f2d2b583ba9dc1fd02a586437f9f0397
```
Comment (ndowens) [inline srcpkgs/zaz/template]: Below checksum add a newline
then add 
CXXFLAGS="-I${XBPS_CROSS_BASE}/usr/include/freetype2"

**Comment by ndowens** [comment]
The review I did, is how to fix the cross-build.

**Comment by pinnaculum** [comment]
Thanks @ndowens, the cross-build is fixed now.

**Pull request merged**